### PR TITLE
Fw core util edm check class transient 

### PR DESCRIPTION
**Pull request by gartung**

### CODE REVIEW

**Comment by gartung** [comment]
@smuzaffar @Dr15Jones

File: FWCore/Utilities/scripts/edmCheckClassTransients [hunk 1]
```diff
@@ -34,25 +34,26 @@ class RMParser(object):
...
+        return 0
     for trans in classtransients[templname]:
       tdm = c.GetDataMember(trans)
       retval = False
       if tdm : retval = tdm.IsPersistent()
-      if retval == True : raise RuntimeError("Error: field '"+trans+"' must be labeled transient=\"true\" in classes_def.xml for '"+name+"'")
-
+      if retval == True : 
+          print "Error: field='"+trans+"' must be labeled transient=\"true\" in classes_def.xml for "+name+" or the equivalent class name with typedefs."
+          return 1
```
Comment (smuzaffar) [inline FWCore/Utilities/scripts/edmCheckClassTransients]: why return here? don't you want to keep looping for the other tansients classes?

Comment (gartung) [inline FWCore/Utilities/scripts/edmCheckClassTransients]: Forgot it was in a loop. I will change it.

**Comment by smuzaffar** [comment]
+1
looks good

**Pull request merged**